### PR TITLE
Update Quiz Documentation

### PR DESCRIPTION
**Pull request by mendelB**
### What does this PR do?

This PR updates our Quiz documentation to -as far as my brain can tell right now- pretty much covers the full quiz feature set.

### What are the relevant tickets/cards?

Refs [Pivotal ID #158443662](https://www.pivotaltracker.com/story/show/158443662)

### CODE REVIEW

File: docs/content-publishing/quiz.md [hunk 1]
```diff
@@ -1,113 +1,196 @@
 # Quiz
 
-The **Quiz** component is a results-based quiz inspired by the _BuzzFeed_ personality quiz format. We use the Quiz to promote campaigns, drive traffic, and engage our members around issues that matter. It's somewhat involved to build correctly, but we'll attempt to keep it as simple and clear as we can. So, as the youths say: let's get quizzin'!
+The **Quiz** component is a results-based quiz inspired by the _BuzzFeed_ personality quiz format.
+
+We use the Quiz to promote campaigns, drive traffic, and engage our members around issues that matter. It's somewhat involved to build correctly, but we'll attempt to keep it as simple and clear as we can.
+
+_So, as the youths say: let's get quizzin'!_
```
Comment (weerd) [inline docs/content-publishing/quiz.md]: 🗣

File: docs/content-publishing/quiz.md [hunk 2]
```diff
@@ -1,113 +1,196 @@
...
 
-The **Results** field is the core result content that you want to display to the user once they've completed the quiz (i.e., filled out all the quiz questions). A piece of result content might be: "Your Beyoncé album is: I Am...Sasha Fierce" for the old school fan. We'd add _all_ the potential results -- namely the full list of the Bey's albums -- each to a new Result item in the **Results** field.
+The **Results** field is the core result content that you want to display to the user once they've completed the quiz \(they complete the quiz by filling out all the quiz questions\). A piece of result content might be:
+
+> "Your Beyoncé album is: I Am...Sasha Fierce"
 
-![Quiz Results Field](../.gitbook/assets/quiz-results-field.png)
+We'd add _all_ the potential results \(which we've already assembled\) -- namely the full list of the Bey's albums -- each to a new Result item in the **Results** field.
+
+![Quiz Results Field](../.gitbook/assets/quiz-results-field%20%281%29.png)
```
[the remark this answers is not in the document]
Comment (mendelB) [reply docs/content-publishing/quiz.md]: Ahh ok I'll repair this

[the remark this answers is not in the document]
Comment (weerd) [reply docs/content-publishing/quiz.md]: 🔧

[the remark this answers is not in the document]
Comment (mendelB) [reply docs/content-publishing/quiz.md]: Ok should be all fixed!

**Pull request merged**